Documentation/_static/helm-reference.css

```diff
@@ -0,0 +1,12 @@
+#helm-reference .wy-table-responsive table td {
+    white-space: normal!important;
+    word-wrap: break-word!important;
+}
+
+#helm-reference .wy-table-responsive table td:nth-child(1) {
+    max-width: 320px!important;
```

Comment:
Maybe make this first column _slightly_ smaller? My concern is that the resulting array _seems_ to just fit with its three columns. But it has a fourth one! that users can easily miss, especially because they have to scroll down _a lot_ to see the horizontal scrolling bar hinting at more content. If we could just see a little bit from the fourth column to understand there's more on the side...?

Reply:
Sure, makes total sense 👍🏼